quantmsutils/diann/diann2mztab.py

```diff
@@ -362,7 +362,7 @@ def diann_version(self) -> str:
         return diann_version_id
 
     def validate_diann_version(self) -> None:
-        supported_diann_versions = ["1.8.1", "1.9.beta.1"]
+        supported_diann_versions = ["1.8.1", "1.9.beta.1", "1.9.2"]
```

Comment:
We could use the output from the versions.yml to see which DIANN is actually installed in the container.
I am a bit worried about people using mismatching versions in container in parameter

Reply:
We are working on this now in the workflow. In this package, we only need to control the supported versions. In the workflow we have to take the version from Diann and passed to the script.